src/inference/src/os/win/win_system_conf.cpp

```diff
@@ -185,10 +185,13 @@ void parse_processor_info_win(const char* base_ptr,
                 }
                 num_blocked++;
             } else if (1 == list_len) {
-                _cpu_mapping_table[list[0] + base_proc][CPU_MAP_CORE_TYPE] = MAIN_CORE_PROC;
-                _cpu_mapping_table[list[0] + base_proc][CPU_MAP_GROUP_ID] = group;
-                _proc_type_table[0][MAIN_CORE_PROC]++;
-                group++;
+                if ((_cpu_mapping_table.size() > list[0]) &&
+                    (_cpu_mapping_table[list[0] + base_proc][CPU_MAP_CORE_TYPE] == -1)) {
```

Comment:
Are these conditions for Win 7 format?

Reply:
So far we only have Win 7 data for Core platform which will not touch this part.